firebase_admin/auth.py

```diff
@@ -430,6 +440,16 @@ def disabled(self):
         """
         return bool(self._data.get('disabled'))
 
+    @property
+    def tokens_valid_after_time(self):
+        """Returns the time before which tokens are invalid.
+
+        Returns:
+            int: Timestap in seconds since the epoch. All tokens issued before that time
+                 are considered revoked.
+        """
+        return int(self._data.get('validSince', 0))
```

Comment:
We seem to return 0 by default.

1. Is that consistent with the behavior in Node SDK (perhaps `None` is more appropriate here)?
2. We should document this in the docstring.